src/hardware/vga_draw.cpp

```diff
@@ -1250,9 +1252,13 @@ static void VGA_VerticalTimer(uint32_t /*val*/)
 	case M_TANDY_TEXT:
 	case M_CGA_TEXT_COMPOSITE:
 	case M_HERC_TEXT:
-		if (machine==MCH_HERC) vga.draw.linear_mask = 0xfff; // 1 page
-		else if (IS_EGAVGA_ARCH) vga.draw.linear_mask = 0x7fff; // 8 pages
-		else vga.draw.linear_mask = 0x3fff; // CGA, Tandy 4 pages
+		if (machine == MachineType::Hercules) {
+			vga.draw.linear_mask = 0xfff; // 1 page
+		} else if (IS_EGAVGA_ARCH) {
```

Comment:
Forgot about this one.

Reply:
I didn't handle this macro yet, the rework is not finished.